modules/configuring-trustyai-with-a-database.adoc

```diff
@@ -85,13 +86,13 @@ stringData:
   databaseName: trustyai_service <7>
 ----
 +
-<1> The database kind, either `mariadb` or `mysql`.
+<1> The only currently supported `databaseKind` value is `mariadb`.
 <2> The username you want TrustyAI to use when interfacing with the database.
-<3> The password for the database username.
-<4> The Kubernetes (K8s) service through which your database is accessed.
-<5> The port the database is listening on.
-<6> The setting for the link:https://quarkus.io/guides/hibernate-orm#quarkus-hibernate-orm_quarkus-hibernate-orm-database-generation[`quarkus.hibernate-orm.database.generation`] argument, which determines how TrustyAI interacts with the database on its initial connection. Set to `none`, `create`, `drop-and-create`, `drop`, `update`, or `validate`. 
-<7> The name of the individual database within the database service that the username and password authenticate to.
+<3> The password that TrustyAi should use when connecting to the database.
+<4> `databaseService` (default, pre-filled value=mariadb) is the Kubernetes (K8s) service that TrustyAI should use when connecting to the database.
+<5> The `databasePort` is given a default, pre-filled value=3306 and is the port that TrustyAI should use when connecting to the database.
```

Comment:
The port that TrustyAI must use when connecting to the database (default is `3306`).